roles/firewall/firewalld/handlers/main.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: Check if firewalld is started
+  service:
+    name: firewalld
+    state: started
+  check_mode: yes
+  register: start_firewalld
+  listen: "Reload firewalld"
+
+- name: Reload firewalld if needed
+  service:
+    name: firewalld
+    state: reloaded
```

Comment:
Why do you use handler to reload ? In TDP we manually launch restart playbook when needed.

Reply:
I did this since reloading firewalld is fast and has no side effects. This is not comparable to restarting a distributed service IMO.

Reply:
When updating `tdp_vars`, `*_config` playbooks and `*_restart` playbooks will be launch so the reload is just for fun here as restart will be done.

Reply:
Removed handlers